deploy_tix/__main__.py

```diff
@@ -15,22 +17,16 @@ def main(args=None):
         formatter_class=argparse.ArgumentDefaultsHelpFormatter)
 
     parser.add_argument(
-        '-r', '--repo',
+        '-o', '--repo-owner',
```

Comment:
rather than owner, I will use organization, but yeah it depends of the context :)